CHANGELOG

```diff
@@ -8,18 +8,638 @@ INIFileEditor - graphical parameter editor for INI files
 Parameters - list of algorithm or TOPP tool parameters that changed in this release
 
 ------------------------------------------------------------------------------------------
-----                                OpenMS 1.12.0                                     ----
+----                                OpenMS 2.0                                        ----
 ------------------------------------------------------------------------------------------
-Release date: unknown
+Release date: January 2015
 
-OpenMS 1.12.0 is the first release after the switch to git and a complete overhaul of the
-build system. Furthermore we removed the dependency to GSL in OpenMS 1.12.0 and replaced
-the functionality with Eigen3 and Wildmagic.
+OpenMS 2.0 is the first release after the switch to git and a complete overhaul of the
+build system. It introduces a considerable number of new features and bug fixes.
+Furthermore, we removed the dependency to GSL and replaced the functionality using
+Eigen3 and Wildmagic.
```

Comment:
state that means that now the core and the full build is non-GPL (e.g. either Apache or BSD) licence